CHANGELOG.md

```diff
@@ -3,6 +3,18 @@ All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Changes
+- renamed event mapping utility function to match naming change to Sensu Go map_go_event_into_ruby
```

Comment:
ideally we should indicate that
>  `map_v2_event_into_v1` was renamed to ...

Also these are breaking changes and should be called out as such.